jetstream/pre_treatment.py

```diff
@@ -55,6 +55,17 @@ def apply(self, df: DataFrame, col: str) -> DataFrame:
         return df.loc[mask, :]
 
 
+@attr.s(auto_attribs=True)
+class CensorLowestValues(PreTreatment):
+    """Removes rows with the lowest n% of values."""
+
+    fraction: float = 1e-5
```

Comment:
Currently, it seems like it's not possible to change this fraction. I guess we should add support for this?